dts/arm/atmel/sam3x.dtsi

```diff
@@ -246,6 +246,15 @@
 			clocks = <&pmc PMC_TYPE_PERIPHERAL 1>;
 			user-nrst;
 		};
+
+		rtc: rtc@400e1a60 {
+			compatible = "atmel,sam-rtc";
+			interrupts = <2 0>;
+			peripheral-id = <2>;
```

Comment:
out of date, replace by clocks. Fix everywhere.

Reply:
updated